src/api-service/__app__/onefuzzlib/orm.py

```diff
@@ -212,6 +214,9 @@ def raw(
         return result
 
 
+Type_has_secrets: Dict[type, bool] = dict()
```

Comment:
I'm not sure what's going on here.

It looks like we have a module-scoped mutable variable. It starts out empty, and when we call `hide_secrets()` on some particular class, the global gets updated.

This seems very tricky. I'm not really sure how I'm supposed to tell if it is correct. Could you talk a little about what's going on here?

Reply:
Less, important, but so I don't forget: stylistically, assuming this is intended to be a private "constant", we'd probably want to name this like:
```
_TYPE_HAS_SECRETS: Dict[type, bool] = dict()
```
Leading `_` to indicate "intended private", all caps to indicate "constant-ish".

Reply:
commented and renamed